include/oneapi/dpl/pstl/utils_ranges.h

```diff
@@ -433,6 +444,7 @@ struct permutation_view_simple<_R, _M, typename ::std::enable_if<is_map_view<_M>
 //permutation discard view:
 struct permutation_discard_view
 {
+    using range_value_t = void; //dummy type
```

Comment:
Should it be `ignore_copyable`?

Reply:
I had intention to write that... But I think it is meaningless to treat with value type of permutation_discard_view or it is not?